### ISSUE TITLE
[docs] auth-service's objects life cycle

### ISSUE THREAD

**Issue by eoln** [mojaloop/project#1943]
## **Goal**:
`As a` Dev

`I want to` see depicted lifecycles for data objects related to Consent/Linking
  
`so that` I will better understand the data model

**Acceptance Criteria**:
- [ ] Consent/Linking lifecycle
- [ ] maybe more...

**Complexity**: <Low> > there are sequence diagrams for use cases, lifecycle description will be complementary documentation 

**Uncertainty**: <Medium> > maybe some internal states for these objects are not covered by sequence diagrams because error scenarios are not covered yet...

---

**Tasks**:
- [ ] - Consent/Linking
- [ ] - 

**Done**
- [ ] Acceptance Criteria pass
- [ ] TBD

## **Pull Requests**:
- [ ] TBD

## **Follow-up**:
- N/A

**Dependencies**:
- N/A

## **Accountability**:
- Owner: TBC
- QA/Review: TBC

**Comment by lewisdaly** [comment]
Thanks @eoln  - this is a good point. I'll mark is as blocked by #1922 for the time being, since the scope of what the Auth Service has changed slightly with the official API Draft.

**Comment by lewisdaly** [comment]
Parking this for now, we are working on technical debt of the auth-service, so we may not need to do it.

**Comment by lewisdaly** [comment]
Out of scope for now. Feel free to reopen if this changes.
